homeassistant/components/bluetooth/__init__.py

```diff
@@ -115,6 +116,15 @@ class BluetoothScanningMode(Enum):
 ]
 
 
+@hass_callback
```

Comment:
This will need docs, but I'm going to defer that until we use it for HKC since we may need to change it at that time. For now the goal was to cleanup the tests without having to access the singleton directly there which makes everything much cleaner.